tests/test_release.py

```diff
@@ -120,20 +121,20 @@ def test_python_version(build: Build, pyproject):
     pyproject_meta = pyproject["tool"]["poetry"]
     wheel_meta = build.wheel.get_meta(version=pyproject_meta["version"])
     python_dependency = pyproject_meta["dependencies"]["python"]
-    m = re.match(r">=(\d+\.\d+),<(\d+\.\d+)", python_dependency)
+    m = re.match(r">=(\d+\.\d+)", python_dependency)
     if m:
-        min_version, max_version = m.groups()
+        min_version, *_ = m.groups()
     else:
         raise ValueError(python_dependency)
 
     min_version_tuple = version_to_tuple(min_version)
-    max_version_tuple = version_to_tuple(max_version)
 
     for classifier in wheel_meta.get_all("Classifier"):
         if classifier.startswith("Programming Language :: Python ::"):
             version_tuple = version_to_tuple(classifier.split("::")[-1].strip())
             if len(version_tuple) > 1:
-                assert version_tuple >= min_version_tuple and version_tuple < max_version_tuple
+                assert version_tuple >= min_version_tuple
```

Comment:
I wonder if this test is really needed. Poetry automatically generates the classifiers and will start supporting the Python 3.12 one when https://github.com/python-poetry/poetry/pull/8397 is shipped.

Reply:
Yes, this test was added when poetry was not adding the classifiers automatically.

This test_release.py is to see if poetry or me or something else changes the wheel and thereby adding somethink under the radar. So I'd keep this test.